README.md

```diff
@@ -1,82 +1,24 @@
 # Trin
-Trin is an Ethereum "portal": a json-rpc server with nearly instant sync, and
-low CPU & storage usage.
 
-Trin does this by making these tradeoffs:
-- Trusts miners to include valid state transitions, to skip sync
-- Shards state across a (new) p2p network, to reduce local storage needs
+Trin is a Rust implementation of a [Portal Network](https://github.com/ethereum/portal-network-specs) client.
 
-This should sound similar to a light client. It is, but with a peer-to-peer
-philosophy rather than the LES client/server model, which has introduced
-challenges in an altruistic environment.
-
-## Repository Structure
-
-The portal protocol is a collection of networks. Trin needs to connect to all of them.
-
-The repository is structured as follows:
-
-- `trin/`: The main entry point to run a holistic Ethereum portal client
-- `trin-history/`: The chain history network
-- `trin-state/`: The state network
-
-## Ready for production?
-
-LOL, not even a little bit. At the last readme update, this was simply a proxy
-to Infura for all inbound requests, and doesn't validate any answers against
-state roots.
-
-Trin will proxy at least *some* requests to Infura for quite a while, but the
-plan is to incrementally reduce the reliance on Infura, as more trin
-functionality becomes available.
+The Portal Network is still in the research phase, so understand that this client is experimental.
```

Comment:
*nitpick* ... "so understand ... " - just feels a bit intense imo

Reply:
Hmm okay, good to know. Then "...so please understand..."? Or what would you recommend?

Reply:
I'd probably merge this line with the `do not use in production` line.... Something like `The Portal Network is under research and active development. Changes to the protocol are likely. Likewise, Trin is currently in alpha and not yet ready to be used in production.` But again, it's just a nitpick